tests/SampleApps/nodejs/azure-pages-sample/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "azure-pages",
+  "version": "",
+  "scripts": {
+    "build": "tsc",
+    "build:production": "npm run prestart && npm prune --production",
+    "watch": "tsc --w",
+    "prestart": "npm run build && func extensions install",
+    "start:host": "func start",
+    "start": "npm-run-all --parallel start:host watch",
+    "test": "echo \"No tests yet...\""
+  },
+  "description": "",
+  "devDependencies": {
```

Comment:
This app does not have prod dependencies but does have dev dependencies. This PR is to fix a bug and enable building this app successfully.